src/software/simulation/er_force_simulator.cpp

```diff
@@ -76,6 +75,50 @@ ErForceSimulator::ErForceSimulator(const TbotsProto::FieldType& field_type,
     this->resetCurrentTime();
 }
 
+std::unique_ptr<RealismConfigErForce> ErForceSimulator::createDefaultRealismConfig()
+{
+    auto realism_config = std::make_unique<RealismConfigErForce>();
+    realism_config->set_stddev_ball_p(0);
+    realism_config->set_stddev_robot_p(0);
+    realism_config->set_stddev_robot_phi(0);
+    realism_config->set_stddev_ball_area(0);
+    realism_config->set_enable_invisible_ball(true);
+    realism_config->set_ball_visibility_threshold(0.4f);
+    realism_config->set_camera_overlap(0.3f);
+    realism_config->set_dribbler_ball_detections(0);
+    realism_config->set_camera_position_error(0);
+    realism_config->set_robot_command_loss(0);
+    realism_config->set_robot_response_loss(0);
+    realism_config->set_missing_ball_detections(0);
+    realism_config->set_vision_delay(0);
+    realism_config->set_vision_processing_time(0);
+    realism_config->set_missing_ball_detections(0);
+    realism_config->set_simulate_dribbling(false);
+    return realism_config;
+}
+
+std::unique_ptr<RealismConfigErForce> ErForceSimulator::createRealisticRealismConfig()
+{
```

Comment:
I'm now curious where these values came from, since they seem to be different than the ones used by default in the ErForce sim: 
https://github.com/UBC-Thunderbots/Software/blob/87bcb4d311addd2d0995cd8d796f30464ab9a6a3/src/extlibs/er_force_sim/src/amun/simulator/simulator.cpp#L151-L165

Reply:
realistic config comes from here https://github.com/robotics-erlangen/framework/blob/master/config/simulator-realism/Realistic.txt